content/300-accelerate/900-compare.mdx

```diff
@@ -0,0 +1,106 @@
+- [**Auto-scaling**](/accelerate/compare#performance): Accelerate automatically adjusts resources to match workload demands, providing fast and consistent performance during traffic spikes.
+
+## Accelerate global cache
+
+Prisma Accelerate offers a powerful global cache, so you can serve data to your users at the edge — the closest point to where the users are located — no matter where your database is hosted. This not only speeds up the experience for users, but also reduces read load on your database as well by avoiding roundtrips.
+
+|  | Accelerate | Hyperdrive | Planetscale Boost |
+| --- | --- | --- | --- |
+| **Fully Managed** | ✅ | ✅ | ✅ |
+| **Globally distributed edge infra** | ✅ | ✅ | ✅ |
+| **Control cache policy from code** | ✅ | ❌ | ❌ |
+| **Query-level cache policies** | ✅ | ❌ | ❌ |
+| **Authenticate with API key** | ✅ | ❌ | ❌ |
+| **Postgres compatible** | ✅ | ✅ | ❌ |
+| **MySQL compatible** | ✅ | ❌ | ✅ |
+| **MongoDB compatible** | ✅ | ❌ | ❌ |
+| **Automatic cache updates** | ❌ | ❌ | ✅ |
+
+**Why are these important?**
+
+- Since Accelerate extends the Prisma client, you can control caching policies directly from your codebase with just an extra line of code. Integration is seamless. Here is an example using [the stale-while-revalidating caching strategy](/accelerate/caching#stale-while-revalidate-swr):
+    
+    ```jsx
+    await prisma.user.findMany({
+      cacheStrategy: {
+        swr: 60,
+      },
+    });
+    ```
+    
+- Query level cache policies are critical for serious applications, so that you can control which queries are cached, and the characteristics of the policy. You may want certain data in your app to be cached for several days, other data to be cached for a just a few minutes, and other data to be not cached at all. This is only possible with Prisma Accelerate.
+- Authenticating with an API key can be a helpful security measure, allowing you to decouple database credentials from application secrets. Easily rotate API keys as often as you like, without needing any credential changes in your database
+- Automatic cache updates means that the cache is automatically updated when a change in the database occurs. With Accelerate, you are in control of how the cache is invalidated, using [various caching strategies](/accelerate/caching). 
+
+
+## Accelerate connection pool
+
+Prisma Accelerate includes a globally hosted connection pooler, which allows you to handle peak loads without any problem. Using a connection pool is important especially for serverless infrastructure, which by nature is not able to control connection volume to the database on its own. Prisma Accelerate offers a fully managed, globally colocated option, which auto scales to support any workload.
+
+### Management
+
+|  | Accelerate | pgbouncer | pgcat | Digital Ocean (pgbouncer) | Neon (pgbouncer) | Supavisor | Hyperdrive |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| **Fully Managed** | ✅ | ❌ | ❌ | 🟠 | ❌ | ❌ | ✅ |
```

Comment:
@sneub, how come the Neon connection pool is marked as not fully managed? 🤔 I'd think it is, but maybe missing something

Reply:
Fixed 👍